modules/core/src/webgl/texture.js

```diff
@@ -29,7 +29,13 @@ export const TEXTURE_FORMATS = {
   [GL.RGBA]: {dataFormat: GL.RGBA, types: [GL.UNSIGNED_BYTE, GL.UNSIGNED_SHORT_4_4_4_4, GL.UNSIGNED_SHORT_5_5_5_1]},
   [GL.ALPHA]: {dataFormat: GL.ALPHA, types: [GL.UNSIGNED_BYTE]},
   [GL.LUMINANCE]: {dataFormat: GL.LUMINANCE, types: [GL.UNSIGNED_BYTE]},
-  [GL.LUMINANCE_ALPHA]: {dataFormat: GL.LUMINANCE_ALPHA, types: [GL.UNSIGNED_BYTE]}
+  [GL.LUMINANCE_ALPHA]: {dataFormat: GL.LUMINANCE_ALPHA, types: [GL.UNSIGNED_BYTE]},
+
+  // 32 bit floats
+  [GL.R32F]: {dataFormat: GL.RED, types: [GL.FLOAT], gl2: true},
```

Comment:
Nit: can these support HALF_FLOAT?

Reply:
We have separate `16F` versions, will add them later as needed.